docs/user/package-management/history-and-rollback.md

```diff
@@ -3,34 +3,57 @@ title: History and Software Rollback
 summary: History and Software Rollback
 ---
 
-# History and Software Rollback
+# History and software rollback in Solus
 
-Solus provides a history and rollback feature via its package manager, `eopkg`. This feature allows you to see when your repository was last updated, actions such as installation, removal, and upgrading of software.
+Solus offers a feature to view your history of software changes and revert your system to previous versions. 
 
-Our rollback feature allows you to essentially rewind your system's software back to a previous state, assuming you or our repository has the versions in question.
+You can use this feature if you have issues after installing packages or system updates, and you need to return to a working state.
 
-## History
+:::warning[Important]
 
-You can see the history from `eopkg` by using:
+The rollback operation only works if:
 
-```bash
-eopkg history
-```
+- The Solus repository has the version of the package you need.
+- You have a local copy of the package.
+
+:::
+
+## History of software changes
+
+- To see your history of software changes, open a terminal and execute:
+
+    ```bash
+    eopkg history
+    ```
+
+    The terminal displays a list of all the software changes in your system.
+
+    ![eopkg history output](./eopkg-history-output.png)
 
 ## Rollback
 
-:::info
+1. Disable any [local repository](https://help.getsol.us/docs/packaging/advanced-config/local-repository#disabling-the-local-solbuild-repository-in-eopkg) you have.
 
-If you have a local repository set up, [disable it](https://help.getsol.us/docs/packaging/advanced-config/local-repository#disabling-the-local-solbuild-repository-in-eopkg) before doing rollbacks or you may get an error.
+    Local repositories can cause errors during rollbacks.
 
-> Error parsing repository index information. Index file does not exist or is malformed.
+1. Open a terminal.
 
-:::
+1. Check the operation history.
+
+    ```bash
+    sudo eopkg history
+    ```
+
+1. Note the number of the operation you want to undo.
+
+1. Do the rollback
+
+    1. Use the following command:
 
-To rollback your system, first use the above history command to check what the transaction / operation number was. Then, we use the following command:
+    ```bash
+    sudo eopkg history -t NUMBER
+    ```
 
-```bash
-sudo eopkg history -t number
-```
+    Replace `NUMBER` with the operation number immediately before the one you want to undo.
 
-The number, in this case, is the operation before the one you want to change. So if the number was `100`, then you would use `99`.
+    For example, if you want to undo operation 100, use 99 as the number.
```

Comment:
The wording of this section is something that has always confused me. Perhaps a better way of describing it is "going back to the desired package state" instead of an operation to undo. So, in the example, if you want to roll back operation 100, the state you want to be on is operation 99.

Reply:
I know what you mean. I updated the wording of the section, let me know what you think about the new wording.